napari/layers/points/points.py

```diff
@@ -784,72 +685,84 @@ def edge_width(self, edge_width: Union[None, float]) -> None:
         self._edge_width = edge_width
         self.events.edge_width()
 
+    @property
```

Comment:
I think we should wait on this for the next PR when we add all the deprecations, but I love it otherwise

Reply:
Removed!